Model/Client.php

```diff
@@ -184,7 +184,10 @@ private function getClient($url, $method = \Zend_Http_Client::GET)
             'useragent' => $this->tjHelper->getUserAgent(),
             'referer' => $this->tjHelper->getStoreUrl()
         ]);
-        $client->setHeaders('Authorization', 'Bearer ' . $this->apiKey);
+        $client->setHeaders(array(
```

Comment:
@dallendalton We can use shortened array syntax here `[]` given Magento's PHP requirements.

Reply:
This is now fixed.